lib/event_icon_builder.rb

```diff
@@ -0,0 +1,36 @@
+class EventIconBuilder
```

Comment:
I remember looking at [`JBuilder`](https://github.com/rails/jbuilder) a while ago and thinking it looked neat